src/sqlparser/src/parser.rs

```diff
@@ -2634,21 +2634,14 @@ impl Parser<'_> {
                 self.next_token();
                 column_inner_field = Some(inner_field);
 
-                if let Token::Word(w) = self.peek_token().token {
-                    match w.keyword {
-                        Keyword::BYTEA => {
-                            header_inner_expect_type = Some(DataType::Bytea);
-                            self.next_token();
-                        }
-                        Keyword::VARCHAR => {
-                            header_inner_expect_type = Some(DataType::Varchar);
-                            self.next_token();
-                        }
-                        _ => {
-                            // default to bytea
-                            header_inner_expect_type = Some(DataType::Bytea);
-                        }
+                match self.parse_data_type() {
```

Comment:
The following SQL is ambiguous according to our current syntax:
```
INCLUDE header 'foo' INCLUDE ...
```
because `INCLUDE` can also be considered the name of a custom data type. In PostgreSQL:
```
test=# select null::include;
ERROR:  type "include" does not exist
LINE 1: select null::include;
                     ^

test=# create table foo (f1 include);
ERROR:  type "include" does not exist
LINE 1: create table foo (f1 include);
                             ^
```

Reply:
Mitigated by disallowing all `DataType::Custom` variants.